modules/build-raptor/src/build-raptor-cli.ts

```diff
@@ -67,8 +104,20 @@ async function run(options: Options) {
   logger.print(`logging to ${logFile}`)
   const isCi = getEnv('CI') === 'true'
   const commitHash = getEnv('GITHUB_SHA')
+
+  let pullRequest: number | undefined
+  try {
+    if (commitHash) {
+      pullRequest = await findPRForCommit(commitHash)
+    }
+  } catch (error) {
+    logger.error(`Failed to find PR for commit ${commitHash}`, error)
```

Comment:
I prefer to fail hard on this - we probably would not notice an error in the log

Reply:
done